src/parser/jobs/blu/index.tsx

```diff
@@ -6,10 +6,11 @@ import React from 'react'
 import {Icon, Message} from 'semantic-ui-react'
 import {changelog} from './changelog'
 
-const description = t('blu.about.description')`This is a limited analyzer for a limited job. It's not ready yet, but please look forward to it.
+const description = t('blu.about.description')`BLU has wildly different play styles depending on your mimickry.  This page will primarily focus on the DPS aspect of the job, so do take the advice with a grain of salt if you are healing or tanking.
 
-This analyzer will focus on DoTs, uptime, and proper usage of [~action/BRISTLE],[~action/MOON_FLUTE], and [~action/SURPANAKHA].
-`
+Worth also keeping in mind the usual adage that BLU content is won and lost on mechanics, not DPS.  Following the suggestions here to optimize your performance will always be welcome, but it should not detract from mechanics; greeding Phantom Flurry ticks at the expense of your team mate's sanity is not recommended.
```

Comment:
Another tiny little thing, teammates'

Reply:
Patched in ff3f05c59604bb1ac24a4e8aa90d275f2920cf0c